compact/tracker.xml

```diff
@@ -158,13 +158,14 @@
     </plugin>
   </plugins>
 
-  <include ref="tracker/vertex_barrel.xml"/>
-  <include ref="tracker/silicon_barrel.xml"/>
-  <include ref="tracker/mpgd_barrel.xml"/>
-  <include ref="tracker/mpgd_dirc.xml"/>
-  <include ref="tracker/silicon_disks.xml"/>
-  <include ref="tracker/support_service_assembly.xml"/>
-  <include ref="tof_barrel.xml"/>
-  <include ref="tof_endcap.xml"/>
+	    <include ref="tracker/vertex_barrel.xml"/>
+	    <include ref="tracker/silicon_barrel.xml"/>
+	    <include ref="tracker/mpgd_barrel.xml"/>
+            <include ref="tracker/mpgd_dirc.xml"/>
+            <include ref="dirc.xml"/>
```

Comment:
Isn't this going to load the DIRC twice?

Reply:
Yes, I had it there to see the dirc with the tracking system. I forgot to remove it before I pushed